src/FakeItEasy/Configuration/IExceptionThrowerConfiguration.cs

```diff
@@ -1,20 +1,77 @@
 namespace FakeItEasy.Configuration
 {
     using System;
+    using System.Diagnostics.CodeAnalysis;

```

Comment:
For the life of me, I can't see where this is used. I could checkout and build and remove it, but so lazy!